### ISSUE TITLE
Alignment is not preserved when auto-correcting by `Layout/SpaceAroundOperators`

### ISSUE THREAD

**Issue by khiav223577** [rubocop-hq/rubocop#8616]
## Expected behavior

The alignment should be preserved.

## Actual behavior

Alignment is preserved only in some lines.

## Steps to reproduce the problem

Add this two files:

### scene_title.rb
```rb
class Scene_Title
  # -------------------------------------------------------------------------
  # ● 戰鬥測試
  # -------------------------------------------------------------------------
  def battle_test
    # 重置測量遊戲時間用的畫面計數器
    Graphics.frame_count = 0
    # 製作各種遊戲目標
    $game_temp          = Game_Temp.new
    $game_system        = Game_System.new
    $game_switches      = Game_Switches.new
    $game_variables     = Game_Variables.new
    $game_self_switches = Game_SelfSwitches.new
    $game_screen        = Game_Screen.new
    $game_actors        = Game_Actors.new
    $game_party         = Game_Party.new
    $game_troop         = Game_Troop.new
    $game_map           = Game_Map.new
    $game_player        = Game_Player.new
    # 切換到戰鬥畫面
    $scene = Scene_Battle.new
  end
end
```

### .rubocop.yml
```yml
Layout/SpaceAroundOperators:
  AllowForAlignment: true
  Enabled: true
```

Run auto-correcting:

```rb
rubocop scene_title.rb --auto-correct --only Layout/SpaceAroundOperators
```

### scene_title.rb (auto-corrected)

```rb
class Scene_Title
  # -------------------------------------------------------------------------
  # ● 戰鬥測試
  # -------------------------------------------------------------------------
  def battle_test
    # 重置測量遊戲時間用的畫面計數器
    Graphics.frame_count = 0
    # 製作各種遊戲目標
    $game_temp = Game_Temp.new
    $game_system = Game_System.new
    $game_switches = Game_Switches.new
    $game_variables = Game_Variables.new
    $game_self_switches = Game_SelfSwitches.new
    $game_screen        = Game_Screen.new
    $game_actors        = Game_Actors.new
    $game_party         = Game_Party.new
    $game_troop         = Game_Troop.new
    $game_map           = Game_Map.new
    $game_player        = Game_Player.new
    # 切換到戰鬥畫面
    $scene = Scene_Battle.new
  end
end
```

## RuboCop version

```
$ [bundle exec] rubocop -V
0.89.1 (using Parser 2.7.1.4, rubocop-ast 0.3.0, running on ruby 2.5.5 x86_64-linux)
```

**Comment by dvandersluis** [comment]
`Layout/SpaceAroundOperators` currently does not see comments as breaks for indentation, and in fact there's a test that would fail otherwise (currently only blank lines are seen as breaks for alignment purposes):

https://github.com/rubocop-hq/rubocop/blob/640e95175d9ef3148c72833368f3d202fab4fdca/spec/rubocop/cop/layout/space_around_operators_spec.rb#L75-L77

So the sample code is not seen as aligned operators, because these two lines are not aligned:

``` ruby
Graphics.frame_count = 0
# 製作各種遊戲目標
$game_temp = Game_Temp.new
```

@koic I was looking into fixing this and I can make it consider comments to be breaks as well, but this will change the behaviour and break tests (for `Layout/ExtraSpacing` as well). So this probably should remain as is. Thoughts?
